src/open_inwoner/accounts/tests/test_profile_views.py

```diff
@@ -346,6 +350,86 @@ def test_expected_form_is_rendered(self):
 
         self.assertEqual(type(form), BrpUserForm)
 
+    def test_image_is_saved_when_begeleider_and_default_login(self):
+        self.user.contact_type = ContactTypeChoices.begeleider
+        self.user.save()
+
+        image = Image.new("RGB", (10, 10))
+        byteIO = io.BytesIO()
+        image.save(byteIO, format="png")
+        img_bytes = byteIO.getvalue()
+
+        response = self.app.get(self.url, user=self.user, status=200)
+        form = response.forms["profile-edit"]
+        form["image"] = Upload("test_image.png", img_bytes, "image/png")
+        form_response = form.submit()
```

Comment:
This block is also showing up often while not very interesting to what we're testing.

Might as well put these together and make a method like `upload_test_image_to_profile_edit_page()` or something.